controller.js

```diff
@@ -0,0 +1,111 @@
+
+const Controller = () => {
+    const moudl = moudle()
+    const vie = view()
+    const addingnewpost = (post) => {
+        moudl.addPost(post)
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const addingnewcomment = (postid, comment) => {
+        moudl.addAcommentToApost(postid, comment)
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const deleteThePost = (id) => {
+        moudl.removeAPost(id)
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const deleteTheComment = (postID, commentID) => {
+        moudl.removeACommentFromAPost(postID, commentID)
+        vie.renderPosts(moudl.getUpdatedPosts())
+    }
+    const addNewUser = (user, pass) => {
```

Comment:
it is ok if your variables are a bit longer use the full word `password`

Reply:
loool ooooch